engine/dlib/src/test/test_transform.cpp

```diff
@@ -15,7 +15,7 @@
 #include "dlib/transform.h"
 #include "dlib/math.h"
 
-using namespace Vectormath::Aos;
+using namespace dmVMath;
```

Comment:
In tests (a single translation unit), it's more convenient to add it at global namespace scope.